ctdconverter/galaxy/converter.py

```diff
@@ -832,9 +832,16 @@ def import_macros(tool, model, **kwargs):
         import_node.text = os.path.basename(macro_file.name)
 
 
-def expand_macro(node, macro):
+def expand_macro(node, macro, attribs=None):
+    """
+    add <expand macro="..." ... /> to node
+    """
```

Comment:
If we were running `pydocstyle` it would complain about this. Shall I add that tool in another PR?

Reply:
In general I'm very positive about this. Just tried it -> ~250 warnings or errors. So this would create quite a burden for future contributors. Maybe we can limit it to changed files?

Reply:
Yes, I can limit the complaints to only changed files: https://github.com/common-workflow-language/cwltool/blob/ab713b6c3bd2c590deb725bd7ce0aab7b2764811/Makefile#L99

Reply:
```
ctdconverter/galaxy/converter.py:836: D200: One-line docstring should fit on one line with quotes (found 3)
ctdconverter/galaxy/converter.py:836: D400: First line should end with a period (not 'e')
ctdconverter/galaxy/converter.py:836: D403: First word of the first line should be properly capitalized ('Add', not 'add')
```